config/gradle/jre.gradle

```diff
@@ -1,46 +1,46 @@
-enum Jre {
-    // TODO Use `.tar.gz` on Linux
-    Linux32("zip"),
-    Linux64("zip"),
-    // TODO rename `Mac` to `MacOS` when artifact is renamed: https://github.com/MovingBlocks/TerasologyJRE/pull/5
-    Mac("zip"),
-    Windows32("zip"),
-    Windows64("zip");
-
-    public URL downloadUrl;
-    public File targetArchive;
-
-    Jre(String extension) {
-        String baseUrl = "http://artifactory.terasology.org/artifactory/libs-release-local"
-
-        // TODO can we use gradle dependencies/configurations to download this for the bundling task?
-        // Use vendored JRE from our Artifactory
-        String groupId = "org/terasology/jre/liberica"
-        String artifactId = this.name().toLowerCase()
-        String version = "0.0.1"
-
-        String artifact = "${artifactId}-${version}.${extension}"
-
-        this.downloadUrl = new URL("${baseUrl}/${groupId}/${artifactId}/${version}/${artifact}")
-        this.targetArchive = new File("${artifact}")   
-    }
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Uses Bellsoft Liberica JRE
+def jreVersion = '8u212'
+def jreUrlBase = "https://download.bell-sw.com/java/$jreVersion/bellsoft-jre$jreVersion"
```

Comment:
The fancier dist work here is all kinds of cool and please go nuts 👍 

This part is one of the reasons I leaned toward separate JRE handling: with this in place we have a hard dependency on a 3rd party URL as part of the launcher build. Not necessarily a big deal if it changes slightly or even disappears, we can certainly recover from that, and we have plenty of other such dependencies, but here is a simple alternative to cache it ourselves to have a reliable target we only have to worry about when we're looking at changing target JRE, not per-build :-)

Also, it means we'd be hitting our own infrastructure instead of theirs to retrieve big stuff, bandwidth-wise. Silly distinction, who really cares about bandwidth these days, but it seems sort of like a nice internet neighbor type thing. May be more of a thing if all our stuff is geographically very close, like within the same Kubernetes cluster.

Reply:
Agreed. I would like to get things like the versioning scheme for our JRE bundles sorted out before, though. 

I hope this will help us sort out things later on.